### PR TITLE
Navigation section links are not translated

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony ready for review :)

File: babycenter/templatetags/babycenter_tags.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from django.template import Library
+
+register = Library()
+
+
+@register.assignment_tag(takes_context=True)
+def load_sections(context):
```
Comment (miltontony) [inline babycenter/templatetags/babycenter_tags.py]: :ok_hand: 
Can we also add this to `molo.core`

**Comment by miltontony** [comment]
:+1:

File: babycenter/templates/base.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% load wagtailcore_tags compress static wagtailuserbar core_tags i18n %}
+{% load wagtailcore_tags compress static wagtailuserbar core_tags i18n babycenter_tags %}
```
Comment (miltontony) [inline babycenter/templates/base.html]: I don't think you need this anymore

**Pull request merged**